### PR TITLE
MARS ROVER DOCUMENTATION-Kindly suggest necessary changes

### PR DESCRIPTION
**Pull request by Pratyush-932**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,128 @@
 # Mars-Rover
 Open Projects 2021
+
+## Abstract
+The **Mars Rover** is a small vehicle built to cover mars like uneven terrains. It is made to cover ueneven surfaces with ease and further to collect and analyze data. It has a robotic arm for data collection process and contains various sensors to recieve and record the data. It also has cameras, lidar and various other sensors for visual data interpretation and navigation.
```
Comment (vanshgoyal) [inline README.md]: Justify the text in readme using paragraph tag. For eg.
`<p align="justify">`
  `<h2>Abstract</h2>`
  `<p>A Rube Goldberg Machine is a contraption intentionally designed to perform a simple task in an indirect and overly complicated way. The aim of this project is to depict the life of a college student through a similar complex assembly of mechanical and electronic components linked with each other creating a metaphor between humans and machines.</p>`
`</p>`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,128 @@
...
+
+
+## Team members
+1. Avinash Singh
+2. Harikrishnan PB
+3. Pratyush Ojha
+4. Sambit Prabhu
+
+## Mentors:
+1. Vansh Goyal
```
Comment (vanshgoyal) [inline README.md]: Hyperlink Github Ids of team members and mentors.

**Review by vanshgoyal — COMMENTED** [review]
Add the research doc to poster and report.

**Comment by theobscuredev** [comment]
@Pratyush-932 Lemme know once you've made the final changes. Make sure to tag me and Vansh (@) in this thread.

**Comment by Pratyush-932** [comment]
@theobscuredev  and @vanshgoyal Sir the final changes have been made. Thankyou

**Comment by Pratyush-932** [comment]
> @Pratyush-932 Lemme know once you've made the final changes. Make sure to tag me and Vansh (@) in this thread.

@theobscuredev and @vanshgoyal Sir the final changes have been made. Thanyou!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,155 @@
 # Mars-Rover
 Open Projects 2021
```
Comment (theobscuredev) [inline README.md]: Apply header as follows:
## Open Projects 2021

File: README.md [hunk 4]
```diff
@@ -1,2 +1,155 @@
...
+[Mars Rover 2020 Perseverance] Body Assembly- https://www.youtube.com/watch?v=McydwwR8xu4  
+JPL Mars Science Laboratory The Curiosity Rover- https://www.youtube.com/watch?v=007SnaUxi40  
+
+
+**Research links-**
+https://www.hindawi.com/journals/ijae/2016/5181097/  
+https://mars.nasa.gov/mars2020/spacecraft/rover/  
+https://llis.nasa.gov/lesson/11501  
+http://robotics.estec.esa.int/ASTRA/Astra2004/Papers/astra2004_D-05.pdf  
+https://teamares.sserd.org/project_files/Ares_MCEV_Report.pdf
```
Comment (theobscuredev) [inline README.md]: Hyperlink the Github IDs and References properly. No need to link the URLs again. Refer this README: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md

**Review by theobscuredev — CHANGES_REQUESTED** [review]
Hey @Pratyush-932, 
Capitalize D in while naming the 'Mechanical Design' folder. And update the README as per changes suggested below.

Just do these final few changes and we're good to go!

File: README.md (hunk 4 above)
Comment (Pratyush-932) [reply README.md]: > Hyperlink the Github IDs and References properly. No need to link the URLs again. Refer this README: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md

Changes made sir.

File: README.md (hunk 3 above)
Comment (Pratyush-932) [reply README.md]: Done @theobscuredev sir.

**Comment by theobscuredev** [comment]
@Pratyush-932 & team,
Make the following changes ASAP and then we'll be good to go with the merge:

1. Justify the entire text in README using the `<p>` paragraph tag and `justify` attribute. Make sure you justify all the paragraphs separately. For eg:
`<p align="justify">`
  `<h2>Abstract</h2>`
  `<p>A Rube Goldberg Machine is .........................</p>`
`</p>`

Refer this README for the same: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md

2. The 'Applications' section in your README is supposed to depict the applications of your robot and not the software used!! Try to mention the applications in the context of the parts and mechanisms used. For eg: You can explain how the actual prototype of the bot will benefit from the use of a particular mechanism/feature in real-life scenario. Refer previous year documentation (For eg: https://github.com/marsiitr/Underwater-Rover#applications) for more info.

**Comment by theobscuredev** [comment]
@Pratyush-932 Also make sure you or someone from you team keeps updating this thread and tags me once the necessary changes have been made!!

**Comment by theobscuredev** [comment]
Hi @Pratyush-932,
Just make this one final change in the README file:
Center align all the images using the `<p>` paragraph tag and `align` attribute. For eg:
`<p align="center">`
  `LED Matrix`
  `<i>LED Matrix</i>`
`</p>`

And add appropriate title for each image.
Also try to resize the images by assigning proper dimensions.

Refer this README for more info: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md

**Comment by theobscuredev** [comment]
@Pratyush-932 Lemme know once you make these final changes and then we'll be good to merge!

**Comment by Pratyush-932** [comment]
> @Pratyush-932 Lemme know once you make these final changes and then we'll be good to merge!

@theobscuredev Sir the changes have been made. I apologize for the late response.

**Comment by theobscuredev** [comment]
@Pratyush-932 Hey Pratyush! I just saw you guys haven't made a cost structure yet. Prepare a tabulated structure for your bot. Make sure you also include the cost of the 3d printed components. Refer the cost structure in this README: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md and include it in your README following the order mentioned in the guidelines.

**Comment by Pratyush-932** [comment]
> @Pratyush-932 Hey Pratyush! I just saw you guys haven't made a cost structure yet. Prepare a tabulated structure for your bot. Make sure you also include the cost of the 3d printed components. Refer the cost structure in this README: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md and include it in your README following the order mentioned in the guidelines.

Done @theobscuredev  Sir

**Comment by Pratyush-932** [comment]
@theobscuredev sir some slight changes have been made to the cost structure.

**Comment by theobscuredev** [comment]
Nice Work!
@SanjeevKrishnan I suggest for merge

**Review by SanjeevKrishnan — APPROVED** [review]
Great work team!
PR looks good. Merging the PR

**Pull request merged**